pkgs/tools/security/wad/default.nix

```
@@ -0,0 +1,46 @@
+{ fetchPypi
+, mock
+, pytestCheckHook
+, python3
+, six
+, stdenv
+}:
```

Calling this package with `python3Packages.callPackage` allows to python access functions and packages directly:

```suggestion
{ lib
, buildPythonApplication
, fetchPypi
, mock
, pytestCheckHook
, six
}:
```

---

Changed